recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
 
 test:
     requires:
-        - python {{ environ['PY_VER'] + '*' }}  # [win]
+        - python {{ PY_VER ~ '*' }}  # [win]
```

Comment:
BTW this is the same fix that @jakirkham did in https://github.com/conda-forge/hdf5-feedstock/pull/13/files, but a different approach. I'd rather if we settle on version only. (I don't have a preferred version though as less typing is good, but that line is quite obscure.)